.github/workflows/container.yml

```diff
@@ -0,0 +1,54 @@
+name: Container
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
```

Comment:
We actually don't have tags for `crates/serverless` releases right now

Reply:
Do you have any plans on releasing it with changeset? If I understand it, changeset could create a tag and this new pipeline could run on this specific tag?

https://github.com/changesets/changesets/blob/8ede784e5fc3629858a57643fdaab76ffb631520/docs/versioning-apps.md

Reply:
Yeah, we could definitely do that. And the above would become something like:

```yml
tags:
  - @lagon/serverless@*
```

Reply:
I have changed the config, so it will use the `@lagon/serverless@...` tag and its version now to push images.